plugins/modal/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@blockly/plugin-modal",
+  "version": "0.20200427.0",
```

Comment:
Use 1.x unless it's in beta.

Reply:
What about leaving this as zero and bump when we publish?

Reply:
Sure.